src/firebolt/async_db/cursor.py

```diff
@@ -199,14 +201,23 @@ async def _do_execute(
                         skip_parsing,
                         async_execution,
                     )
-                    response = await self._api_request(
-                        query,
-                        {
-                            "async_execution": 1,
-                            "advanced_mode": 1,
-                            "output_format": JSON_OUTPUT_FORMAT,
-                        },
-                    )
+
+                    with Timer() as runTime:
+                        response = await self._api_request(
+                            query,
+                            {
+                                "async_execution": 1,
+                                "advanced_mode": 1,
+                                "output_format": JSON_OUTPUT_FORMAT,
+                            },
+                        )
+
+                    envVar = environ.get("FIREBOLT_SDK_PERFORMANCE_DEBUG", "0")
+                    if envVar == "1":
+                        logger.debug(
+                            f"[PERFORMANCE] Running query {query[:50]} ... {runTime}s"
```

Comment:
Do not add dots `...` in case the query length is less then 50 characters. It will look weird then